src/citrine/gemtables/columns.py

```diff
@@ -57,16 +56,10 @@ class Column(PolymorphicSerializable['Column']):
     Abstract type that returns the proper type given a serialized dict.
     """
 
-    @abstractmethod
-    def _attrs(self) -> List[str]:
-        pass  # pragma: no cover
-
     def __eq__(self, other):
-        try:
-            return all([
-                self.__getattribute__(key) == other.__getattribute__(key) for key in self._attrs()
-            ])
-        except AttributeError:
+        if isinstance(other, Serializable):
+            return self.dump() == other.dump()
+        else:
             return False
```

Comment:
This pattern is generally repeated.  Not fundamentally opposed to dropping the isinstance test for a try instead on the potential `AttributeError`.

Reply:
Meh, this is fine by me.